lib/replicator.js

```diff
@@ -2156,6 +2165,12 @@ module.exports = class Replicator {
     for (const peer of this.peers) peer._resetMissingBlock(index)
   }
 
+  get hasFullCore () {
```

Comment:
fullyDownloaded () { is more in line with the rest of your naming imo and make it a non getter

Reply:
done, and renamed the one on Peer to `fullyDownloadedSignaled`